dlp/api/DlpSample/RiskAnalysis.cs

```diff
@@ -0,0 +1,501 @@
+        public string RegionCode { get; set; }
+    }
+
+    public class RiskAnalysis
+    {
+        static object NumericalStats(NumericalStatsOptions opts) {
+            DlpServiceClient dlp = DlpServiceClient.Create();
+
+            // Construct + submit the job
+            RiskAnalysisJobConfig config = new RiskAnalysisJobConfig
+            {
+                PrivacyMetric = new PrivacyMetric {
+                    NumericalStatsConfig = new NumericalStatsConfig {
+                        Field = new FieldId { Name = opts.ColumnName }
+                    }
+                },
+                SourceTable = new BigQueryTable {
+                    ProjectId = opts.TableProjectId,
+                    DatasetId = opts.DatasetId,
+                    TableId = opts.TableId
+                }
+            };
+            config.Actions.Add(new Google.Cloud.Dlp.V2.Action
+            {
+                PubSub = new PublishToPubSub{
+                    Topic = opts.TopicId
+                }
+            });
+
+            var submittedJob = dlp.CreateDlpJob(new CreateDlpJobRequest {
+                Parent = $"projects/{opts.CallingProjectId}",
+                RiskJob = config
+            });
+
+            // Listen to pub/sub for the job
+            SubscriptionName subscriptionName = new SubscriptionName(opts.CallingProjectId,
+    opts.SubscriptionId);
+            SubscriberClient subscriber = SubscriberClient.Create(
+                subscriptionName, new[] { SubscriberServiceApiClient.Create() });
+            // SimpleSubscriber runs your message handle function on multiple
+            // threads to maximize throughput.
+            ManualResetEvent done = new ManualResetEvent(false);
+            subscriber.StartAsync((PubsubMessage message, CancellationToken cancel) =>
+            {
+                if (message.Attributes["DlpJobName"] == submittedJob.Name)
+                {
+                    done.Set();
+                    return Task.FromResult(SubscriberClient.Reply.Ack);
+                }
+                else
+                {
+                    return Task.FromResult(SubscriberClient.Reply.Nack);
+                }
+            });
+
+            done.WaitOne();
+            subscriber.StopAsync(CancellationToken.None).Wait();
+
+            // Process results
+            var resultJob = dlp.GetDlpJob(new GetDlpJobRequest
+            {
+                DlpJobName = DlpJobName.Parse(submittedJob.Name)
+            });
+
+            var result = resultJob.RiskDetails.NumericalStatsResult;
+
+            Console.WriteLine($"Value Range: [{result.MinValue}, {result.MaxValue}]");
+            string lastValue = "";
+            for (int quantile = 0; quantile < result.QuantileValues.Count; quantile++)
+            {
+                string value = result.QuantileValues[quantile].ToString(); // TODO print this nicer
```

Comment:
please, rename the variable. There is a keyword in C# called "value" that means a value of a property